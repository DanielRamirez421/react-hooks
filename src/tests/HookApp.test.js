import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe('HookApp component tests', () => {

  test('should matchSnapShot with component', () => {
    let wrapper = shallow( <HookApp /> );
    expect( wrapper ).toMatchSnapshot();
  });
  

});
