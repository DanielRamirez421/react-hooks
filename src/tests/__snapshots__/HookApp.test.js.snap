// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp component tests should matchSnapShot with component 1`] = `
<div>
  <h1>
    Hello World!
  </h1>
</div>
`;
